resources/terms-of-use.mdwn

```diff
@@ -38,7 +38,7 @@ You not only release Status’ liability on a number of items, but also agree th
 
 - Arbitration
 If a dispute arises between you and Status, we will seek to first solve it amicably with you before proceeding to the use of an arbitration administered by the Swiss Chambers’ Arbitration Institution.
-&nbsp;
+
```

Comment:
Here we're removing these `&nbsp;` because they're rendered as plain text in the app